lib/parser/response.rb

```diff
@@ -1,21 +1,49 @@
+require 'cgi'
+require 'beta_code'
+
 require_relative './word'
 require_relative './unknown'
 
 module Parser
+  CREATED_DATE = '2020-01-01T00:00:00.000000'
+
   class Response
-    def initialize(doc)
+    def initialize(word, latin, doc)
+      @word = CGI.escapeHTML(word)
+      @latin = latin
+
       @words = doc.xpath('./words/word').map { |w| Word.new(w) }
       @unknowns = doc.xpath('./words/unknown').map { |u| Unknown.new(u) }
     end
 
-    def to_xml
-      @to_xml ||= "<words>\n#{components.map(&:to_xml).join}</words>"
+    def bamboo_xml
+      @bamboo_xml ||= <<~XML
+        <rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+          <oac:Annotation xmlns:oac="http://www.openannotation.org/ns/" rdf:about="#{morphology_service}">
+            <dcterms:creator xmlns:dcterms="http://purl.org/dc/terms/">
+              <foaf:Agent xmlns:foaf="http://xmlns.com/foaf/0.1/" rdf:about="org.perseus:tools:morpheus.v1"/>
+            </dcterms:creator>
+            <dcterms:created xmlns:dcterms="http://purl.org/dc/terms/">#{CREATED_DATE}</dcterms:created>
```

Comment:
This is one of the two places where this code returns a different result from the Morphsvc code. Instead of using the current date for `dcterms:created` it uses the string `2020-01-01T00:00:00.000000`. The [documentation](https://dublincore.org/specifications/dublin-core/dcmi-terms/) says that it should be the "Date of creation of the resource." But generating a date makes comparing results and caching more difficult.

I was thinking that instead of having a new date every time the resource is accessed, we would update the `CREATED_DATE` whenever there was a change in the code that could lead to a different result.